src/js/RateCurve.js

```diff
@@ -165,23 +164,23 @@ RateCurve.prototype.rcCommandRawToDegreesPerSecond = function (rcData, rate, rcR
 
         const rcCommandfAbs = Math.abs(rcCommandf);
 
-        switch(TABS.pid_tuning.currentRatesType) {
-            case TABS.pid_tuning.RATES_TYPE.RACEFLIGHT:
+        switch (FC.RC_TUNING.rates_type) {
+            case FC.RATES_TYPE.RACEFLIGHT:
```

Comment:
I'm not too sure about this change, it implies that we now use the FC rates and not the ones in the TAB? The idea is to show the model preview and can change it live in the TAB. Are we modifying the FC value while the user changes the TAB values?
In either way, I think the best way is to pass an object to the method with the values that we must to represent. Use the FC makes this library dependent on the FC file and is not necessary.
Maybe the `currentRates` object that returns the new method?

Reply:
Fixed here (as we use this in two tabs:

![image](https://user-images.githubusercontent.com/8344830/161149373-f54c61ec-f31e-4698-a27d-4465026b9745.png)

Reply:
It's also fixes changing rate type and updating currentRates,